packages/core-components/src/components/paragraph/paragraph.scss

```diff
@@ -34,4 +33,16 @@
   &--align-center {
     text-align: center;
   }
+
+  &--variant-black-100 {
+    color: var(--b2b-color-copy-default);
```

Comment:
shall we make this use black-100

Reply:
I'd like to stick with the default copy color, so that when we change that in the future, all paragraphs will change. Plus it was like this in the previous version of the component.